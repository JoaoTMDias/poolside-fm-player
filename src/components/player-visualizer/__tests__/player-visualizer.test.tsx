import React from "react";
import { shallow } from "enzyme";

import PlayerVisualizer from "components/player-visualizer/index";

describe("<PlayerVisualizer />", () => {
	it("should render a visualizer", () => {
		const component = shallow(<PlayerVisualizer />);

		expect(component).toMatchSnapshot();
	});

	it("should add a is-playing class when there is media playing", () => {
		const component = shallow(<PlayerVisualizer isPlaying />);
		expect(component.hasClass("player-visualizer--is-playing")).toBe(true);
	});
});
