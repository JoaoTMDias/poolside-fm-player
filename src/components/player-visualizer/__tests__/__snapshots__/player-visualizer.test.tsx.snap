// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlayerVisualizer /> should render a visualizer 1`] = `ShallowWrapper {}`;
